CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Pasting into clients only supporting `UTF8_STRING` mime type on Wayland
 - Crash when copying/pasting with neither pointer nor keyboard focus on Wayland
 - Crash due to fd leak on Wayland
+- CLI arguments ignored without a configuration file present
```

Comment:
It's not a bug on 0.5.0, since it's from recent options rework, so it should be removed.